subsys/bluetooth/controller/hci_driver.c

```diff
@@ -235,6 +241,7 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_BT_PERIPHERAL) ||
 #define MEMPOOL_SIZE ((PERIPHERAL_COUNT * PERIPHERAL_MEM_SIZE) + \
 		      (SDC_CENTRAL_COUNT * CENTRAL_MEM_SIZE) + \
 		      (SDC_ADV_SET_MEM_SIZE) + \
+		      (SDC_LE_POWER_CONTROL_SIZE) + \
```

Comment:
```suggestion
		      (SDC_LE_POWER_CONTROL_MEM_SIZE) + \
```
just to be consistent